lib/galaxy/managers/hdas.py

```diff
@@ -275,6 +275,7 @@ def __init__(self, app: MinimalManagerApp):
             'url',
             'create_time',
             'update_time',
+            'tool_type',
```

Comment:
Check performance, possibly just drop from this view if we can get away with only betawebclient view?